src/ray/gcs/CMakeLists.txt

```diff
@@ -30,7 +30,18 @@ add_custom_command(
   TARGET gen_gcs_fbs
   COMMAND ${FLATBUFFERS_COMPILER} -p -o ${PYTHON_OUTPUT_DIR} ${GCS_FBS_SRC}
   DEPENDS ${FBS_DEPENDS}
-  COMMENT "Running flatc compiler on ${GCS_FBS_SRC}"
+  COMMENT "Running flatc compiler on ${GCS_FBS_SRC} for python"
+  VERBATIM)
+
+# Generate Java bindings for the flatbuffers objects.
+set(RAY_HOME ${CMAKE_CURRENT_LIST_DIR}/../../..)
+set(JAVA_OUTPUT_DIR ${RAY_HOME}/java/runtime/src/main/java/org/ray/runtime/generated/)
+add_custom_command(
+  TARGET gen_gcs_fbs
+  COMMAND ${FLATBUFFERS_COMPILER} -j -o ${JAVA_OUTPUT_DIR} ${GCS_FBS_SRC}
+  COMMAND python ${RAY_HOME}/auto_gen_tool.py ${RAY_HOME}
```

Comment:
Let's put the script under `java/` dir, and name it `modify_generated_java_flatbuffers_files.py`